registration-processor/pre-processor/registration-processor-quality-classifier-stage/src/main/java/io/mosip/registration/processor/quality/classifier/stage/QualityClassifierStage.java

```diff
@@ -390,6 +395,15 @@ private iBioProviderApi getBioSdkInstance(BiometricType biometricType) throws Bi
 	
 	private Map<String, String> getQualityTags(List<BIR> birs) throws BiometricException{
 		
+		Map<String, String> tags = new HashMap<String, String>();
+		
+		if (birs == null) {
+			tags.put(qualityTagPrefix.concat("Iris"), biometricNotAvailableTagValue);
+			tags.put(qualityTagPrefix.concat("Finger"), biometricNotAvailableTagValue);
+			tags.put(qualityTagPrefix.concat("Face"), biometricNotAvailableTagValue);
+			return tags;

```

Comment:
don't hard code the modalities in the code, get it from the config, if a country support only 2 modalities then they will update the config to only those 2.

also there is a case of 1 modality alone is not available in the below birs, in that case also we will get into situation where tag is not available. So end of this method also check for modalities that is not available and mark them as biometricNotAvailableTagValue

Reply:
removed hardcode values adn get it from config.... also handled in case modality not available in BIR...